src/app/notebox/page.tsx

```
@@ -0,0 +1,75 @@
+//     </div>
+//   );
+// };
+
+// export default NoteBox;
+// components/NotePage.tsx
+'use client';
+import { useState, useEffect } from 'react';
+import FilterModal from '@/components/noteBoxComponents/FilterModal';
+import FilterBar from '@/components/noteBoxComponents/FilterBar';
+import { useUserData } from '@/hooks/useMyPageQueries';
+import useUserNote from '@/hooks/noteHooks/useUserNotes';
```

ESLint

[error] 'useUserNote' is defined but never used.

(@typescript-eslint/no-unused-vars)